### PR TITLE
add s

### PR DESCRIPTION
**Pull request by Denowich**

### CODE REVIEW

**Comment by Denowich** [comment]
Eugene please review my html and css. Thank you

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+
+let m1 = 'item';
+let m2 = 'activeS'
+let musS = `${style.m1} ${style.m2}`
 
 const Navbar = () => {
   return (
     <nav className={style.nav}>
-      <div className={style.item}>
+      <div className={`${style.item} ${style.active}`}>
```
Comment (Eugen263) [inline src/components/Navbar/Navbar.jsx:16]: the incomprehensible nesting level of elements

**Review by Eugen263 — COMMENTED** [review]
All code reviewed. Make changes please)

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Denowich) [reply src/components/Navbar/Navbar.jsx:16]: I was trying to change the color of items in the navbar component.

File: src/components/Navbar/Navbar.module.css [hunk 2]
```diff
@@ -6,3 +6,11 @@
 .item {
   color: whitesmoke;
```
[the remark this answers is not in the document]
Comment (Denowich) [reply src/components/Navbar/Navbar.module.css:7]: Ok

**Pull request merged**